addons/stock/models/stock_picking.py

```diff
@@ -1020,7 +1020,7 @@ def _create_backorder(self):
                     body=_('The backorder <a href=# data-oe-model=stock.picking data-oe-id=%d>%s</a> has been created.') % (
                         backorder_picking.id, backorder_picking.name))
                 moves_to_backorder.write({'picking_id': backorder_picking.id})
-                moves_to_backorder.mapped('package_level_id').write({'picking_id':backorder_picking.id})
+                moves_to_backorder.mapped('move_line_ids').mapped('package_level_id').write({'picking_id':backorder_picking.id})
```

Comment:
you can remove the `mapped` for relational fields. It makes the code more readable